tests/test_examples.py

```diff
@@ -56,6 +56,8 @@
 ACCURACY_PERF_FACTOR = 0.99
 # Trainings should last at most 5% longer than the baseline
 TRAINING_TIME_PERF_FACTOR = 1.05
+# Evaluations should last at most 5% longer than the baseline
+EVAL_TIME_PERF_FACTOR = 1.05
```

Comment:
I think we can just have a single `TIME_PERF_FACTOR` instead of one for training and one for inference.

Reply:
Done.